requirements/production.txt

```diff
@@ -23,4 +22,5 @@ django-anymail==0.10
 # --------------------------
 raven==6.1.0
 
-
+# custom packages
+python-dateutil
```

Comment:
Did you run into issues without this package? If so, would guess that this would be a base dependency, rather than a production-specific dependency.

Reply:
I only got the missing module error when I ran with production settings, so I wasn't sure if it was just because of the changes to the libraries because production uses gunicorn and a bunch of other services.  If its safe to put into base, then I can move it there.